File: nelio-alves/algoritmo-e-logica-de-programacao/c/exercicios/troco.c
#include <stdio.h>

int main(){
    double precoUnitario, dinheiroRecebido, troco;
    int qtd;

    printf("Preco unitario do produto: ");
    scanf("%lf", &precoUnitario);

    printf("Quantidade comprada: ");
    scanf("%d", &qtd);

    printf("Dinheiro recebido: ");
    scanf("%lf", &dinheiroRecebido);

    troco = dinheiroRecebido - (precoUnitario * qtd);

    printf("TROCO = %.2lf\n", troco);

    return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/java/src/exercicios/Troco.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class Troco {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		double precoUnitario, dinheiroRecebido, troco;
		int qtd;

	    System.out.print("Preco unitario do produto: ");
		precoUnitario = sc.nextDouble();

	    System.out.print("Quantidade comprada: ");
	    qtd = sc.nextInt();

	    System.out.print("Dinheiro recebido: ");
		dinheiroRecebido = sc.nextDouble();

		troco = dinheiroRecebido - (precoUnitario * qtd);

	    System.out.printf("TROCO = %.2f\n", troco);

		sc.close();
	}
}
